CHANGELOG.rst

```diff
@@ -1,6 +1,6 @@
 Version 2.1.6
 -------------
-
+ * `@fiennyangeln <http://github.com/fiennyangeln>`_: Add woff2 (`#3586 <https://github.com/rtfd/readthedocs.org/issues/3586>`_)
```

Comment:
No need to edit the changelog here, this is done _automatically_ (or almost #3524)

Reply:
ok. I will change it

Reply:
Changed! thanks